src/main/java/uk/gov/ons/ssdc/caseprocessor/utils/Constants.java

```diff
@@ -2,5 +2,5 @@
 
 public class Constants {
   public static final String EVENT_SCHEMA_VERSION = "v0.2_RELEASE";
-  public static final String TOPIC_SAMPLE = "rm-internal-sample";
+  public static final String INBOUND_NEW_CASE_TOPIC = "event_new-case";
```

Comment:
this constant is only used by tests, so it should live in `TestConstants`

Reply:
It was previously used [here](https://github.com/ONSdigital/ssdc-rm-caseprocessor/blob/57787d7fb7f62fa25af8a19e4e7c5f0ebbdf7993/src/main/java/uk/gov/ons/ssdc/caseprocessor/messaging/NewCaseReceiver.java#L85) but might more sense to just hardcode it?

Reply:
shouldn't be needed anywhere except integration tests